### PR TITLE
Pause and Resume implemented

### LINKED ISSUE #40: Pause and Resume
So I have found a good feature to be added here...
I thought of adding pause and resume recording to this....
This will a nice one to add I guess

So I would like to work on this feature

### PR DESCRIPTION
**Pull request by Villain45**
fixes #40 
The feature has been implemented as of #40 
So now user can pause and resume whenever required

### CODE REVIEW

**Comment by Villain45** [comment]
@heysagnik add accepted tag bro....
After that you can merge later

**Comment by heysagnik** [comment]
Some changes I would request @Villain45:
1) Remove the border of the svg icons that you have newly added.
Couldn't understand this - When you pause , should it stop the screensharing ?

**Comment by Villain45** [comment]
these borders were initially present for stop button
If you want i can remove them

**Comment by heysagnik** [comment]
> these borders were initially present for stop button If you want i can remove them

yeah and can you please resolve the conflicts

**Comment by heysagnik** [comment]
I asked to remove this border not the one you removed

**Comment by heysagnik** [comment]
> Couldn't understand this - When you pause , should it stop the screensharing ?
If someone wants to record another tab then if he closes the screensharing then a new pop up should be shown so that they can select the new window

did you looked into this ?

File: src/scss/components/recorder.scss [hunk 1]
```diff
@@ -400,7 +400,6 @@
               background-image: url('/src/images/stop.svg');
               min-width: unset;
               padding: 2.5rem;
-              border: 3px solid $white;
```
Comment (heysagnik) [inline src/scss/components/recorder.scss]: add these again

**Review by heysagnik — CHANGES_REQUESTED** [review]

**Comment by Villain45** [comment]
can we have chat in discord? about that

**Review by heysagnik — APPROVED** [review]

**Pull request merged**